types/test/test_interpolation.c

```diff
@@ -150,7 +150,7 @@ static rage_Error time_checks(rage_Interpolator * interpolator) {
     if (frame_no_check(v, 1))
         return RAGE_ERROR("Mismatch at t=4");
     rage_interpolated_view_advance(v, 120);
-    if (frame_no_check(v, 121))
+    if (frame_no_check(v, 1))
         return RAGE_ERROR("Mismatch at t=124");
```

Comment:
If you've changed the above line to have `1`, does this message change to `4`?

Reply:
Don't think so, the message tells you the time the error occurs, not the expected value at that time (which comes out elsewhere).

Reply:
Ah, ok. I wasn't quite sure what `frame_no_check` was taking.